src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,24 @@
-export const TodoInfo = () => {};
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+interface TodoInfoProps {
+  todo: Todo;
+}
+
+export const TodoInfo = ({ todo }: TodoInfoProps) => {
+  const { id, title, completed, user } = todo;
```

Comment:
The destructuring of `user` from `todo` assumes that `user` is always defined. Ensure that the `Todo` type allows for `user` to be `null` or handle this case appropriately to avoid potential runtime errors.